examples/gh-runner-gke-dind/sample-manifests/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: runner-dind-deployment
+spec:
+  selector:
+    matchLabels:
+      app: runner
+  replicas: 2
```

Comment:
Suggest allowing this to be configured by user, with a default value.

Reply:
Now that we are using kustomize, they can override this if needed.